embark.el

```diff
@@ -849,35 +862,40 @@ If NO-DEFAULT is t, no default value is passed to `completing-read'."
 
 ;;; Verbose action indicator
 
-(defvar embark--vindicator-prompt-buffer "*Embark Actions*"
-  "Buffer used by embark-verbose-indicator to display actions and keybidings.")
+(defvar embark--vindicator-prompt-buffer " *Embark Actions*"
+  "Buffer used by `embark-verbose-indicator' to display actions and keybidings.")
 
-(defvar embark--vindicator-display-alist
-  `((,(regexp-quote embark--vindicator-prompt-buffer)
-     (display-buffer-at-bottom)
-     (window-parameters (mode-line-format . none))
-     (window-height . fit-window-to-buffer)))
-  "Parameters added to display-buffer-alist for displaying the actions buffer.")
+(defvar embark--vindicator-display-action
+  '(display-buffer-reuse-window)
+  ;;'(display-buffer-in-side-window (side . right))
+  ;;'(display-buffer-below-selected (window-height . 15))
+  "Parameters added to `display-buffer-alist' for displaying the actions buffer.")
 
 (defvar embark--vindicator-excluded-commands
-  '(embark-collect-snapshot embark-collect-live embark-export
+  '("\\`embark-collect-" embark-export
     embark-keymap-help embark-become embark-isearch nil)
-  "Commands not displayed by embark-verbose-indicator.")
+  "Commands not displayed by `embark-verbose-indicator'.")
 
 (defun embark--vindicator-key-str (k)
-  "A string describing the given key or key code."
+  "A string describing the given key K or key code."
   (if (numberp k) (single-key-description k) (key-description k)))
 
+;; TODO describe DESCS X PREFIX
 (defun embark--vindicator-bind-desc (descs x prefix)
-  "Accumulator used by `embark--vindicator-keymap-descriptor', q.v."
+  "Accumulator used by `embark--vindicator-keymap-descriptor'."
   (let ((k (car x)) (c (cdr x)))
     (cond ((keymapp c)
            (let* ((k-str (embark--vindicator-key-str k))
                   (cds (embark--vindicator-keymap-descriptor c k-str)))
              (cons (max (or (car cds) 0) (or (car descs) 0))
                    (cons (max (or (cadr cds) 0) (or (cadr descs) 0))
                          (append (cddr descs) (cddr cds))))))
-          ((memq c embark--vindicator-excluded-commands) descs)
+          ((seq-find (lambda (x)
+                       (if (symbolp x)
+                           (eq c x)
+                         (string-match-p x (symbol-name c))))
+                     embark--vindicator-excluded-commands)
+           descs)
```

Comment:
i would perhaps define an embark--vindicator-excluded-command-p predicate
to make this code obvious, but it's just a very minor thing.

Reply:
Good idea, added a helper function.